scripts/gis-material-update/validate-deploy/modules/gis_validate_deploy.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from abc import ABC, abstractmethod
+from modules.common import *
```

Comment:
Using wildcard imports is generally discouraged. They make it hard to see where a particular function is defined and can easily cause name clashes.

You can either import the methods one-by-one:
```suggestion
from modules.common import validate_data_count_limits
from modules.common import deploy
```

Or use common as a module, and call the methods with `common.deploy(` etc.
```suggestion
import modules.common as common
```
Since common is a module, maybe it should be used as one?

Reply:
Changed to this:
from modules.common import validate_data_count_limits, deploy